code/modules/almayer/weaponhits.dm

```diff
@@ -16,6 +22,7 @@
  * * salvo - identifies it as a salvo or not.
  */
 /proc/weaponhits(weaponused, location, point_defense = FALSE, salvo = FALSE)
+	var/logtotalhits
 
 
 	switch(weaponused)
```

Comment:
everything in this file (weapons, what they do for damage, etc.) should 100% be datums